bin/docker-image-tool.sh

```diff
@@ -102,33 +114,37 @@ function build {
     error "Failed to build Spark JVM Docker image, please refer to Docker build output for details."
   fi
 
-  docker build $NOCACHEARG "${BINDING_BUILD_ARGS[@]}" \
-    -t $(image_ref spark-py) \
-    -f "$PYDOCKERFILE" .
-    if [ $? -ne 0 ]; then
-      error "Failed to build PySpark Docker image, please refer to Docker build output for details."
+  if [ "${PYDOCKERFILE}" != "skip" ]; then
+    docker build $NOCACHEARG "${BINDING_BUILD_ARGS[@]}" \
+      -t $(image_ref spark-py) \
+      -f "$PYDOCKERFILE" .
+      if [ $? -ne 0 ]; then
+        error "Failed to build PySpark Docker image, please refer to Docker build output for details."
```

Comment:
Script should bail out here, i.e. `exit` if the `docker build` fails

Reply:
Hi @rvesse .. script does exit with non-zero error code when error "..." is called. See error function at top of the file. It echos the message and exits with exit code 1. So indeed this script will fail in case of docker build failures.